### PR TITLE
[WIP] meta: show diffs on pr renders

### PR DESCRIPTION
**Pull request by devsnek**
Like all good things, this uses a bunch of code from whatwg's process.

### CODE REVIEW

File: scripts/diff.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const path = require('path');
+const cp = require('child_process');
+const { JSDOM } = require('jsdom');
+
+const BUILD_PATH = path.resolve(__dirname, '../out/index.html');
+const DIFF_PATH = path.resolve(__dirname, './htmldiff.pl');
+const OLD_PATH = path.resolve(os.tmpdir(), './old.html');
+const NEW_PATH = path.resolve(os.tmpdir(), './new.html');
+
+const last = cp.execSync('git show origin/gh-pages:index.html', {
```
Comment (ljharb) [inline scripts/diff.js]: `origin` isn't necessarily the remote name; mine's `source` (`origin` is my personal fork)

File: spec.html [hunk 2]
```diff
@@ -777,6 +777,7 @@ <h1>Throw an Exception</h1>
         </emu-alg>
         <p>mean the same things as:</p>
         <emu-alg>
+          1. Return *null*.
```
Comment (ljharb) [inline spec.html]: (reminder not to merge this)

File: scripts/diff.js (hunk 1 above)
Comment (devsnek) [reply scripts/diff.js]: i'm not sure what to do. just `git show gh-pages:index.html` isn't enough.

Comment (devsnek) [reply scripts/diff.js]: I came up with `execSync('git branch -a | grep "gh-pages"').toString().trim()`, which works great locally, but fails on netlify

**Comment by ljharb** [comment]
The diff looks very funky: https://deploy-preview-1689--ecma262-snapshots.netlify.com/#sec-ordinaryownpropertykeys

**Comment by jmdyck** [comment]
(Also, it appears to be rendering each `<code>` element on a separate line.)

**Pull request closed without merging**